region/crateria/east.json

```diff
@@ -991,13 +991,39 @@
           "name": "West Ocean Junction (North-South Transition)",
           "nodeType": "junction",
           "nodeSubType": "junction"
+        },
+        {
+          "id": 15,
+          "name": "G-Mode Morph Junction (South West)",
+          "nodeType": "junction",
+          "nodeSubType": "junction",
+          "note": "Represents being at the Moat door with G-Mode and morph or artificial morph"
+        },
+        {
+          "id": 16,
+          "name": "G-Mode Morph Overloaded PLMs Junction (South West)",
+          "nodeType": "junction",
+          "nodeSubType": "junction",
+          "note": "Represents being at the Moat door with G-Mode, PLMs overloaded, and morph or artificial morph"
+        },
+        {
+          "id": 17,
+          "name": "G-Mode Morph Junction Direct (South West)",
+          "nodeType": "junction",
+          "nodeSubType": "junction",
+          "note": "Represents being at the Moat door with G-Mode and morph or artificial morph, in direct G-Mode"
         }
       ],
       "obstacles": [
         {
           "id": "A",
           "name": "Shoot Respawning Super Block",
           "obstacleType": "abstract"
+        },
+        {
+          "id": "B",
+          "name": "G-Mode Overload PLMs with the Item",
```

Comment:
Unclear if this is by powerbombing it or by collecting it.  But it'd have to be powerbombing it.

Reply:
The strats that use this obstacle PB the item. I didnt make the name longer since it was just an obstacle, but I could

Reply:
Now I'm not sure what this obstacle shows as a concept.  It's an ordinary way to overload PLMs isn't it?